modules/installation-ibm-cloud-regions.adoc

```diff
@@ -40,29 +40,24 @@ ifdef::ibm-power-vs[]
 * `dal` (Dallas, USA)
 ** `dal10`
 ** `dal12`
-* `us-east` (Washington DC, USA)
-** `us-east`
 * `eu-de` (Frankfurt, Germany)
 ** `eu-de-1`
 ** `eu-de-2`
-* `lon` (London, UK)
-** `lon04`
-** `lon06`
-* `osa` (Osaka, Japan)
-** `osa21`
+* `mad` (Madrid, Spain)
+** `mad02`
+** `mad04`
 * `sao` (Sao Paulo, Brazil)
 ** `sao01`
```

Comment:
we might not get sao01 in before GA. Let's leave it off the docs for now (sorry for not updating you)